ghstack/config.py

```diff
@@ -44,15 +44,12 @@ def read_config(*, request_circle_token: bool = False) -> Config:  # noqa: C901
         config_path = "/".join([str(current_dir), ".ghstackrc"])
         if os.path.exists(config_path):
             break
+        else:
+            config_path = None
         current_dir = current_dir.parent
 
-    if config_path is None:
-        raise RuntimeError(
-            "Couldn't locate config in parent directories of {local_path}"
-            .format(local_path=os.getcwd()),
-        )
-
-    config.read(['.ghstackrc', os.path.expanduser(config_path)])
+    if config_path is not None:
+        config.read(['.ghstackrc', os.path.expanduser(config_path)])
```

Comment:
OK, so I'm rereviewing this code for real this time (I only did a very cursory review earlier) and I'm confused by this logic:

1. You retained the `expanduser` call, but there is no `~` anywhere to be found, so this seems pointless now...
2. But that means I never ever check for ghstackrc in the home directory! That seems bad. We should fall back to that if it isn't anywhere in the parent directories.
3. You don't need the `.ghstackrc` (current dir) test because it's handled by config path

So I think the more logical logic here is:

1. Scan for ghstackrc (as you've done here)
2. If you don't find it anywhere, fallback to testing for `os,path.expanduser('~/.ghstackrc')`

Reply:
> But that means I never ever check for ghstackrc in the home directory! That seems bad. We should fall back to that if it isn't anywhere in the parent directories.

I was under assumption that people develop somewhere in a subfolder of a home directory. But I see that it might be not a case.